submissions/EnF1NeeT/HTML-CSS_popup/index.html

```diff
@@ -0,0 +1,155 @@
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Settings.svg" alt="Settings" class="icon"> Settings</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/logout.svg" alt="Logout" class="icon"> Logout</a></li>
+                </ul>
+              </div>
+            </div>
+          </li>
+          <li class="logo"><h1>Kottanspost</h1></li>
+          <li class="greeting"><h2>Hi, Kottan!</h2><img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt=""></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="posts">
+        <div class="post">
+          <div class="topzone">
+            <img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt="">
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post1.jpg" class="post-img" alt="">
+          </div>
+          <div class="action-zone">
+            <div class="btn">
+              <img src="./img/favorite_FILL0_wght400_GRAD0_opsz48.png" alt="like">
+              Like
+            </div>
+            <div class="btn">
+              <img src="./img/forum_FILL0_wght400_GRAD0_opsz48.png" alt="comment">
+              Comment
+            </div>
+            <div class="btn">
+              <img src="./img/send_FILL0_wght400_GRAD0_opsz48.png" alt="share">
+              Share
+            </div>
+          </div>
+          <div class="post-time">posted 5h ago</div>
+        </div>
+        <div class="post">
+          <div class="topzone">
+            <img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt="">
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post2.jpg" class="post-img" alt="">
+          </div>
+          <div class="action-zone">
+            <div class="btn">
+              <img src="./img/favorite_FILL0_wght400_GRAD0_opsz48.png" alt="like">
+              Like
+            </div>
+            <div class="btn">
+              <img src="./img/forum_FILL0_wght400_GRAD0_opsz48.png" alt="comment">
+              Comment
+            </div>
+            <div class="btn">
+              <img src="./img/send_FILL0_wght400_GRAD0_opsz48.png" alt="share">
+              Share
+            </div>
+          </div>
+          <div class="post-time">posted 5h ago</div>
+        </div>
+        <div class="post">
+          <div class="topzone">
+            <img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt="">
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post3.jpg" class="post-img" alt="">
+          </div>
+          <div class="action-zone">
+            <div class="btn">
+              <img src="./img/favorite_FILL0_wght400_GRAD0_opsz48.png" alt="like">
+              Like
+            </div>
```

Comment:
Add `cursor: pointer` to all interactive elements.

Reply:
Fixed in the last commit